### PR TITLE
MongoDB class

### PR DESCRIPTION
**Pull request by kabaros**
Intro to DBs, MongoDB, Querying Mongo and connecting with Node

**Note:** If you are adding a new lesson or have edited the name of a lesson file. Please make sure you update `./SUMMARY.md` to reflect the changes in the gitbook contents.

### CODE REVIEW

**Comment by kabaros** [comment]
still WIP .. but it would be good to have feedback especially on https://github.com/kabaros/syllabus/blob/scotland/node-db/lesson4.md

**Comment by Michael-Antczak** [comment]
The only comment I would have for the moment is to make sure you explain all the new terms and not assume that they understand. To me those would be: 
- index
- concurent
- scalling in terms of DB

File: node-db/lesson4-old.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+### Local development
+
+Make sure you install all of the project dependencies with `npm i` before you do anything else. Then you can run these scripts:
+
+* `npm test` - runs ESLint followed by the end-to-end tests
+* `npm start` - runs the server via nodemon, restarting on changes to the `src` directory
+
+Note that this project requires **Node.js 6 or above** due to the user of ES6.
```
Comment (djgrant) [inline node-db/lesson4-old.md]: "use of"

File: node-db/lesson6.md [hunk 2]
```diff
@@ -32,6 +32,15 @@ myFirstPromise.then((successMessage) => {
 
 > **Exercise**: Let's rewrite the code above using `fetch` which provides an alternative to `XmlHttpRequest` that uses a Promise-based API - https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
 
+
+
+# Using Express.Router
+We have lots of Routes in **server.js**, we can use the express.Router class to create modular, mountable route handlers. A Router instance is a complete middleware and routing system; for this reason, it is often referred to as a “mini-app”.
```
Comment (djgrant) [inline node-db/lesson6.md]: A bit jargony. How about

> We have lots of Routes in **server.js**. To better organise them, we can use the `express.Router` class. A `Router` instance is...

**Review by djgrant — APPROVED** [review]

**Pull request merged**